src/bundle/Resources/config/services/persistence.yaml

```diff
@@ -0,0 +1,9 @@
+services:
```

Comment:
Shouldn't this config reside in some more meaningful file like: `services/search.yaml` or `services/query.yaml`?